src/base/index.scss

```diff
@@ -1,5 +1,4 @@
 @import '../support/index.scss';
-
```

Comment:
I kinda liked that the import that doesn't output any styles is somewhat visually separated. But we can always add a comment to separate it. So the stylelint rule should still be fine to keep.